FirstTableViewController.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  FirstTableViewController.swift
+//  ModelModel
+//
+//  Created by Stanislav Cherkasov on 12.12.17.
+//  Copyright © 2017 Stanislav Cherkasov. All rights reserved.
+//
+
+import UIKit
+
+class FirstTableViewController: UITableViewController {
```

Comment:
choose more appropriate name, e.g. UsersViewController, StudentsViewController etc instead of FirstTableViewController